src/flowgger/decoder/rfc3164_decoder.rs

```diff
@@ -63,6 +52,74 @@ struct Pri {
     severity: Option<u8>,
 }
 
+fn decode_rfc_standard(pri: &Pri, msg: &str, line: &str) -> Result<Record, &'static str> {
+    // Decoding "usual" rfc input as advised in the rfc: [<pri>]<datetime> <hostname> <message>
```

Comment:
Usual is a bit of a weighed word here, recommended is probably better

Reply:
fair enough. will update the comment